embedding/cpp/flutter_engine.cc

```diff
@@ -12,19 +12,20 @@
 
 namespace {
 
-void ParseEngineArgs(std::vector<std::string>* list) {
+std::vector<std::string> ParseEngineArgs() {
+  std::vector<std::string> engine_arguments;
```

Comment:
For local consistency:
```suggestion
  std::vector<std::string> engine_args;
```
It can also be named `result` or something so that the reader can better understand the meaning.